src/js/tech/ChromecastTech.js

```diff
@@ -156,6 +157,7 @@ ChromecastTech = {
           mediaInfo = new chrome.cast.media.MediaInfo(source.src, source.type),
           title = this._requestTitle(source),
           subtitle = this._requestSubtitle(source),
+          customData = this._requestCustomData(source) || {},
```

Comment:
Does this need to have an empty object as a default? Doing so prevents `requestCustomDataFn` from intentionally sending a falsey value to `customData`. For example, a user of this plugin might want to send an empty string via `customData` (I'm not quite sure why they would need to do this, but I'd seen crazier things).

Reply:
Fair enough. I'll even take it a step further by only setting `mediaInfo.customData` if the user has passed `requestCustomDataFn`